schedule/functional/extra_tests_gnome_sdk.yaml

```diff
@@ -0,0 +1,9 @@
+name:   extra_test_gnome_sdk
+description:    >
+    Maintainer: jrauch
+    Extra tests about software in sdk on gnome
+schedule:
```

Comment:
I see that at the last module executed for extra tests scenarios is *console/coredump_collect*.
I think we should include it here too.
e.g.: https://openqa.suse.de/tests/3796732#step/coredump_collect/4